test/Feature/StructureSpec.hs

```diff
@@ -452,6 +453,10 @@ spec = do
                     "format": "enum_menagerie_type",
                     "type": "string"
                   },
+                  "arr": {
+                    "format": "text[]",
+                    "type": "string"
```

Comment:
So I had to add those lines to make the tests pass, because I added the array to the `varied_arguments` function. The result does not seem correct, though - I think this looks like #1405. Now I don't think it makes sense to keep this in here like this, because this is not what we want it to do.

If we just changed that here accordingly - did I accidently create a test for https://github.com/PostgREST/postgrest/issues/1405#issuecomment-551325528 ?